ThingIFSDK/ThingIFSDK/ThingIFAPI.swift

```diff
@@ -6,17 +6,33 @@
 import Foundation
 
 /** Class provides API of the ThingIF. */
-open class ThingIFAPI: NSObject, NSCoding {
+open class ThingIFAPI<ConcreteAlias: Alias>: NSObject, NSCoding {
+
+    private class var SHARED_NSUSERDEFAULT_KEY_INSTANCE: String {
```

Comment:
@yongpingchen 
The value is constant. so it is no need to instance property.

I changed class property to static property. I think this is more clear.